content/releases/3-6/5_breaking-changes/release-guide.txt

```diff
@@ -55,8 +55,27 @@ We know that breaking changes suck, when they break your plugins or other parts
 - Direct URLs for the users view have changed, e.g. now `/panel/users/?role=yourRole`
 
 ### Custom Panel views
-- Using custom Panel views (incl. routing, breadcrumbs...) in plugins has changed fundamentally. You will have to use the [new Panel areas feature](/releases/3.6/features/fiber#area-definition) now.
+- Using custom Panel views (incl. routing, breadcrumbs...) in plugins has changed fundamentally. You will have to use the [new Panel areas feature](/releases/3.6/fiber/areas#area-definition) now.
 - Accordingly, the frontend extension `panel.plugin({ view: [...] })` has been  removed. Use Panel areas instead.
+- If the custom panel view is for authenticated users, you will have to wrap your custom view with `<k-inside>`. If not, you should wrap it `<k-outside>`.. [See more component changes](/releases/3.6/breaking-changes#panel__vue-component-changes)
```

Comment:
panel => Panel
you should wrap it `<k-outside>`.. => you should wrap it with `<k-outside>`.

Reply:
Thanks, done.